metric-example/metric-expressions-for-k8s.md

```diff
@@ -46,15 +46,19 @@ builtin:host.cpu.usage:avg
 ```
```

Comment:
Hi @stefan-moschinski-dynatrace - thanks for also taking care of k8s expressions. I've just started yesterday updating this entire thing to our new MINT metrics here (https://github.com/floriangeigl/dynatrace-api/blob/master/metric-example/metric-expressions-for-k8s.md). Can we exclude this file from this PR so I don't have to resolve the merge conflict later? :)
From what I understand, I can remove most () in my expressions. I can take care of this in my fork.